authorize.go

```diff
@@ -44,6 +53,13 @@ func (a *autographer) authorize(r *http.Request, body []byte) (userid string, au
 	}
 	hawk.MaxTimestampSkew = a.auths[userid].hawkMaxTimestampSkew
 	err = auth.Valid()
+	if a.stats != nil {
+		skew := abs(auth.ActualTimestamp.Sub(auth.Timestamp))
+		sendSkewErr := a.stats.Histogram("hawk_timestamp_skew_ns", float64(skew.Nanoseconds()), nil, 1.0)
```

Comment:
Might want to rename this. I'm not sure what to rename it to.

Also, we might want to track whether it exceeded the skew error or not. As a counter or tag?